test/local/account_routes.js

```diff
@@ -1307,49 +1141,12 @@ describe('/account/login', function () {
         assert.ok(!response.verified, 'response indicates account is not verified')
         assert.equal(response.verificationMethod, 'email', 'verificationMethod is email')
         assert.equal(response.verificationReason, 'signup', 'verificationReason is signup')
-      }).then(function () {
-        mockMailer.sendVerifyCode.reset()
-        mockDB.createSessionToken.reset()
-      })
-    })
-
-    describe('sign-in with unverified account', function () {
-      before(() => {
-        mockDB.emailRecord = function () {
-          return P.resolve({
-            authSalt: crypto.randomBytes(32),
-            data: crypto.randomBytes(32),
-            email: 'test@mozilla.com',
-            emailVerified: false,
-            kA: crypto.randomBytes(32),
-            lastAuthAt: function () {
-              return Date.now()
-            },
-            uid: uid,
-            wrapWrapKb: crypto.randomBytes(32)
-          })
-        }
-      })
-
-      it('sends verify account email', function () {
-        return runTest(route, mockRequest, function (response) {
-          assert.equal(mockMailer.sendVerifyCode.callCount, 1, 'mailer.sendVerifyCode was called')
-          assert.equal(mockMailer.sendVerifyLoginEmail.callCount, 0, 'mailer.sendVerifyLoginEmail was not called')
-          assert.equal(mockMailer.sendNewDeviceLoginNotification.callCount, 0, 'mailer.sendNewDeviceLoginNotification was not called')
-          assert.equal(response.verified, false, 'response indicates account is unverified')
-          assert.equal(response.verificationMethod, 'email', 'verificationMethod is email')
-          assert.equal(response.verificationReason, 'signup', 'verificationReason is signup')
-          assert.equal(response.emailSent, true, 'email not sent')
-        }).then(function () {
-          mockMailer.sendVerifyCode.reset()
-        })
```

Comment:
I apologize in advance for the pain involved in reviewing these test changes....